### PR TITLE
Allow extensions to plugin to topology view

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Adding the ability to plugin to the topology view.

Adds plugin capabilities for:

- [x] Component Factories
- [x] Data model transforms
- [x] Create Connectors
- [x] Watched Resources
- [x] Remove model conversion, create model objects in transforms
- [x] Filters

Holding for a future PR:
- [ ] Side Panel content
- [ ] Create items

### CODE REVIEW

File: frontend/packages/console-plugin-sdk/src/registry.ts [hunk 1]
```diff
@@ -101,6 +103,10 @@ export class ExtensionRegistry {
     return this.extensions.filter(isOverviewCRD);
   }
 
+  public getOverviewResourceUtils() {
```
Comment (rawagner) [inline frontend/packages/console-plugin-sdk/src/registry.ts:106]: registry is deprecated. Please use withExtensions HOC or useExtension hook

**Comment by christianvogt** [comment]
Hit a blank screen when I loaded Jai's serverless test ns:
image

I think he uses a setup script:
image

**Comment by jeff-phillips-18** [comment]
> Hit a blank screen when I loaded Jai's serverless test ns:
Fixed, thanks.

**Pull request merged**